src/Bundle/Controller/ViewHandler.php

```diff
@@ -13,16 +13,16 @@
 
 namespace Sylius\Bundle\ResourceBundle\Controller;
 
+use FOS\RestBundle\View\ConfigurableViewHandlerInterface;
 use FOS\RestBundle\View\View;
-use FOS\RestBundle\View\ViewHandler as RestViewHandler;
 use Symfony\Component\HttpFoundation\Response;
 
 final class ViewHandler implements ViewHandlerInterface
 {
-    /** @var RestViewHandler */
+    /** @var ConfigurableViewHandlerInterface */
     private $restViewHandler;
 
-    public function __construct(RestViewHandler $restViewHandler)
+    public function __construct(ConfigurableViewHandlerInterface $restViewHandler)
```

Comment:
It's a slight BC break, but since this PR bumps up the requirement for FOSRestBundle I don't think there's any case one would pass RestViewHandler to the constructor.

Reply:
Yes and it's a final class. So we can't override it.

Reply:
@pamil I am wondering why is it a bc break? Cause this service is marked as final and the old RestViewHandler argument implements ConfigurableViewHandlerInterface.
If we inject the old one with dependency injection, it continues to work.

Reply:
Theoretically, someone could create their own service definition where they inject the original service into it.

Reply:
Yes but IMHO it should work too, cause I replaced with the implemented interface. Inject the old service definition should work too.

Reply:
Ah, that's perfect then! Sorry, I must've missed it.

Reply:
Do we still need this mention in UPGRADE doc?

Reply:
No need then :)